python/idsse_common/idsse/common/publish_confirm.py

```diff
@@ -170,15 +193,24 @@ def stop(self):
         Starting the IOLoop again will allow the publisher to cleanly
         disconnect from RabbitMQ.
         """
-        logger.info('Stopping')
+        logger.info('Stopping Thread %s', self._thread and self._thread.name)
         self._stopping = True
         self._close_connection()
         self._stopping = False  # done stopping
 
+    @property
+    def _is_connected(self) -> bool:
+        """True if RabbitMQ Connection and Channel exist and are open"""
+        return (self._connection is not None and self._connection.is_open
+                and self._channel is not None and self._channel.is_open)
 
+    def _is_running(self) -> bool:
```

Comment:
I'm not sure that I could for this one, since the underlying code (`Thread.is_alive()`) is type `Callable[[None], bool]` instead of simply `bool`. 

Can the `property` tag do some magic to invoke a function and get the resulting True/False when you access the property?